spec/PlayerSpec.js

```diff
@@ -0,0 +1,67 @@
+describe("Player", function() {
+  var playerOne;
+
+  beforeEach(function() {
+     playerOne = new Player("steve");
+  })
+  it(" should be able to store the player name", function() {
+    expect(playerOne.name).toEqual("steve");
+  });
+
+  it(" should be able to store the player's bowls", function() {
```

Comment:
It feels like you are testing the constructor here... Is reading `bowl` a behaviour of your class or just part of the implementation?